charts/px-central/values.yaml

```diff
@@ -249,3 +249,103 @@ images:
     repo: portworx
     imageName: memcached
     tag: 1.6.15-alpine
+
+## Configuration for cortex
```

Comment:
this config is kept as part of pxcentral-cortex secret.
Compare the previous configuration.